<!--REQUEST_ID=22ec2812-ad75-4186-aa92-578213ece87d-->
Failure on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.48683&attempt_id=7d2d801d-9d2d-4794-a4c9-79c6750c1c28)

Attempted: znc, zncModules

<details><summary>Partial log (click to expand)</summary><p>

```
source root is source
patching sources
configuring
no configure script, doing nothing
building
Building "push.so" for ZNC 1.7.1... /nix/store/vcc4svb8gy29g4pam2zja6llkbcwsyiq-binutils-2.30/bin/ld: cannot find -lz
collect2: error: ld returned 1 exit status
[ !! ] Error while building "push.so"
builder for '/nix/store/i2qvvbynmsvrk16n9wxy2h8dw84g64x1-znc-push-git-2016-10-12.drv' failed with exit code 1
error: build of '/nix/store/b7d7fimrhxmmq9xh5xhms6c6lk8s1a1r-znc-backlog-git-2017-06-13.drv', '/nix/store/cinqmyqlb5lvfzil3xlg2csh5hwskxmi-znc-clientbuffer-git-2015-08-27.drv', '/nix/store/cxwgk6mwcjx5nsqc3bnsyc0r0vychkij-znc-privmsg-git-2015-02-22.drv', '/nix/store/i2qvvbynmsvrk16n9wxy2h8dw84g64x1-znc-push-git-2016-10-12.drv', '/nix/store/ja6vx5jhk2q0qfqfh28hvyc9c4c61lnf-znc-playback-git-2015-08-04.drv', '/nix/store/y95c32rk24m6v6zwk8f0zxq4il498qva-znc-fish-git-2014-10-10.drv' failed
```
</p></details>

--- a/pkgs/applications/networking/znc/modules.nix
+++ b/pkgs/applications/networking/znc/modules.nix
@@ -9,6 +9,8 @@ let
     inherit buildPhase;
     inherit installPhase;
 
+    buildInputs = znc.buildInputs;
+
     meta = a.meta // { platforms = stdenv.lib.platforms.unix; };
     passthru.module_name = module_name;
   });

